### ISSUE TITLE
Connect GCP with FLINT UI for GCM sample runs

### ISSUE THREAD

**Issue by iamrajiv** [moja-global/FLINT-UI#118]
I am trying sample GCBM runs locally. I have hosted `FLINT.Cloud` on GCP using Terraform script and got ingress URL as this `https://fc-ingress-4gtzbngbgq-uc.a.run.app/`. So now in FLINT-UI (Vue app) we are using the docker container of GCBM to run the simulations with `http://localhost:8081/` URL. So I replaced `http://localhost:8081/` to `https://fc-ingress-4gtzbngbgq-uc.a.run.app/` all over the `FLINT-UI` codebase thought it will work and we will be able to simulate with GCP as backend but it is not working showing network error. So help is needed here.

**Comment by Simpleshell3** [comment]
Hello @iamrajiv. Can you send a screenshot or error log of the problem.

**Comment by iamrajiv** [comment]
Hi @Simpleshell3 when I try to create new simulation its saying this

```shell
Access to XMLHttpRequest at 'https://fc-ingress-4gtzbngbgq-uc.a.run.app/gcbm/new' from origin 'http://127.0.0.1:8000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
gcbm.js?a32b:674 Error: Network Error
    at createError (createError.js?2d83:16)
    at XMLHttpRequest.handleError (xhr.js?b50d:99)
```

**Comment by Simpleshell3** [comment]
hello @iamrajiv , sorry it's taking some time to address your problem. I wanted to replicate your error by setting up GCP but my card is not supported so it's a little slow.  
 However in the meantime, what I understand is you are doing an XMLHttpRequest to GCP. So the browser is blocking it as it usually allows a request in the same origin for security reasons. You need to do something different when you want to do a cross-domain request. A tutorial about how to achieve that is Using [CORS](http://www.html5rocks.com/en/tutorials/cors/).

If you have set up Flint.cloud correctly, you should have this [file](https://github.com/moja-global/FLINT.Cloud/blob/master/local/rest_api_gcbm/app.py) in your local directory.  Try adding your [ingress URL
](https://fc-ingress-4gtzbngbgq-uc.a.run.app/) to the file and try again. A sample guide on handling CORS policies can be found [here](https://flask-cors.readthedocs.io/en/latest/).

**Comment by iamrajiv** [comment]
I and @Simpleshell3 debugged the CORS problem. We debug CORS by disabling CORS in the browser but still, it is not working.

I guess the problem is related to CORS but the code like I have replaced all GCBM local URL to ingress URL so `new simulation` is working fine and endpoints are hit I was able to see the API graph in the GCP dashboard but in the second step i.e. `Select a simulation` from the list the lists is not showing means the new simulation title is not been saved to GCP like we are not able to fetch the list of simulation from GCP because it is not been saved.
